src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisConfigForm.tsx

```diff
@@ -245,127 +265,221 @@ export const AutomatedAnalysisConfigForm: React.FC<AutomatedAnalysisConfigFormPr
     return '';
   }, [template]);
 
+  const targetSelect = React.useMemo(() => {
+    return (
+      editing && (
+        <StackItem>
+          <TargetSelect simple />
```

Comment:
This should only returns the `<TargetSelect simple />`. The `StackItem` can be wrapped outside.